bot/utils/randomization.py

```diff
@@ -1,6 +1,7 @@
 import itertools
 import random
-import typing as t
+from collections.abc import Iterable
```

Comment:
Don't you need to change some code according to this?

Previously the import was `t.Iterable` yet you never change the code that uses it.